ui/pi-web-agent-app/src/app/live-info/live-info.component.ts

```diff
@@ -8,24 +8,24 @@ import { SystemInfo, SystemInfoService } from '../system-info.service';
 })
 export class LiveInfoComponent implements OnInit {
 
-  constructor(private system_info_service: SystemInfoService) { }
+  constructor(private systemInfoService: SystemInfoService) { }
```

Comment:
I kind of want to keep snake case, makes everything more readable, maybe it's just me getting old lol

Reply:
although in js I'm kind of doing the opposite lol.
If we find an npm formatting tool let's add it

Reply:
at least npm lint can spot it